packages/core/client/src/schema-component/antd/upload/Upload.tsx

```diff
@@ -106,13 +106,14 @@ function AttachmentListItem(props) {
 
   const item = [
     <span key="thumbnail" className={`${prefixCls}-list-item-thumbnail`}>
-      {file.imageUrl && (
+      {file.imageUrl && !file.videoUrl && (
```

Comment:
Better to abstract this property into a more general one, something like `previewUrl`. And also use a `previewType` property to diff them.

Reply:
OK. thanks for your comment, I'll fix it later